sched/worker/workers/makers.go

```diff
@@ -19,7 +19,7 @@ func MakeInmemoryWorker(node cluster.Node, tmp *temp.TempDir) runner.Service {
 // Makes a worker suitable for using as an in-memory worker.
 func MakeDoneWorker(tmp *temp.TempDir) runner.Service {
 	ex := execers.NewDoneExecer()
-	r := runners.NewSingleRunner(ex, snapshots.MakeInvalidFiler(), nil, runners.NewNullOutputCreator(), tmp)
+	r := runners.NewSingleRunner(ex, snapshots.MakeInvalidFiler(), nil, runners.NewNullOutputCreator(), tmp, nil)
```

Comment:
I think its inconsequential - we still need to check for nil in downstream uses in case we add/update code and nil sneaks back in.